pkg/diagnose/diagnose.go

```diff
@@ -131,31 +117,27 @@ func (c *Collector) collectPodDescribe(pod *corev1.Pod) {
 	}
 }
 
-// CollectPodLogs info
-func (c *Collector) CollectPodLogs(corePodSelector labels.Selector) {
-	corePodList := &corev1.PodList{}
-	currentPod := strings.Split(corePodSelector.String(), "=")[0]
-	if !util.KubeList(corePodList, &client.ListOptions{Namespace: options.Namespace, LabelSelector: corePodSelector}) {
-		return
-	}
-	if len(corePodList.Items) == 0 {
-		c.log.Printf(`❌ No %s pods found\n`, currentPod)
+// CollectPodsLogs collects logs of all exsisting pods
+func (c *Collector) CollectPodsLogs() {
+	// List all pods
+	podList := &corev1.PodList{}
+	if !util.KubeList(podList, &client.ListOptions{Namespace: options.Namespace}) {
```

Comment:
> 2 - Instead of manually specifying for which pods logs should be
> collected, logs are automatically collected for all pods (if in the
> future additional pods will be added, logs will be collected for those
> as well without requiring changes)

Is this mean that in an openshift cluster with OCS it will collect all pods logs regardless if they are noobaa related or not? 
In that case, I think it is not the desired outcome.

Reply:
Agree, we cannot assume that our pods are the only ones in the NS

Reply:
Added a pod selector to select only noobaa pods (assuming all noobaa pods have the "app=noobaa" label)